test/src/integration/basic.test.ts

```diff
@@ -63,6 +105,268 @@ describe("solo - 1 node", () => {
         expect(invoice).toEqual({ success: true });
     });
 
+    describe("Sending invalid parcels over the limits (general)", () => {
+        let parcelEncoded: any[];
+        beforeEach(async () => {
+            const nonce = await node.sdk.rpc.chain.getNonce(address);
+            const parcel = node.sdk.core
+                .createPaymentParcel({
+                    recipient,
+                    amount: 0
+                })
+                .sign({
+                    secret,
+                    fee: 10,
+                    nonce
+                });
+            parcelEncoded = parcel.toEncodeObject();
+        });
+
+        test.each(["0x01" + "0".repeat(64), "0x" + "f".repeat(128)])(
+            "nonce: %p",
+            async (nonce, done) => {
+                parcelEncoded[0] = nonce;
+                try {
+                    await node.sendSignedParcelWithRlpBytes(
+                        RLP.encode(parcelEncoded)
+                    );
+                    done.fail();
```

Comment:
It's a common practice to return here.

return done.fail()

Reply:
Moved `done()` into the `catch`.